nilearn/datasets/func.py

```diff
@@ -79,17 +79,21 @@ def fetch_haxby(
     data : sklearn.datasets.base.Bunch
         Dictionary-like object, the interest attributes are :
 
-        - 'anat': string list. Paths to anatomic images.
-        - 'func': string list. Paths to nifti file with :term:`BOLD` data.
-        - 'session_target': string list.
+        - 'anat': :obj:`list` of :obj:`str`. Paths to anatomic images.
+        - 'func': :obj:`list` of :obj:`str`.
+          Paths to nifti file with :term:`BOLD` data.
+        - 'session_target': :obj:`list` of :obj:`str`.
           Paths to text file containing run and target data.
-        - 'mask': string. Path to fullbrain mask file.
-        - 'mask_vt': string list. Paths to nifti ventral temporal mask file.
-        - 'mask_face': string list. Paths to nifti ventral temporal mask file.
-        - 'mask_house': string list. Paths to nifti ventral temporal mask file.
-        - 'mask_face_little': string list.
+        - 'mask': :obj:`str`. Path to fullbrain mask file.
+        - 'mask_vt': :obj:`list` of :obj:`str`.
           Paths to nifti ventral temporal mask file.
-        - 'mask_house_little': string list.
+        - 'mask_face': :obj:`list` of :obj:`str`.
+          Paths to nifti ventral temporal mask file.
+        - 'mask_house': :obj:`list` of :obj:`str`.
+          Paths to nifti ventral temporal mask file.
+        - 'mask_face_little': :obj:`list` of :obj:`str`.
+          Paths to nifti ventral temporal mask file.
+        - 'mask_house_little': :obj:`list` of :obj:`str`.
           Paths to nifti ventral temporal mask file.
```

Comment:
Are these descriptions correct?

Reply:
actually they were too vague but the dataset descriptions has better ones so I copied them over